ui/src/components/AppDetailedView.js

```diff
@@ -1,14 +1,21 @@
 import React from 'react';
+import { Title } from '@patternfly/react-core';
 import Header from './common/Header';
+import AppDetailsContainer from '../containers/AppDetailsContainer';
+import './AppDetailedView.css';
 
 class AppDetailedView extends React.Component {
   render () {
     return (
       <div className="app-detailed-view">
         <Header />
+        <Title className="title" size="2xl">
```

Comment:
I went with 3xl for the Header on the Landing Page.

Reply:
yes, from the flat, there is an App ID header that will live above this `Deployed Versions` one, so my thinking is that one should be 3x1, and this one 2x1

Reply:
Resolving this for now as per above